en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+
+## Model assertions that allow adding a system user
+
+### The simplest case is where you did everything:
+
+* You created a snapcraft key and it is registered to your SSO
+* You created a model assertion
+* The model assertion has the `authority-id` set to your SSO account ID
+* The model assertion has the `brand-id` set to your SSO account ID
+* You signed the model assertion using the key mentioned above
+* You built the image using that model assertion
+
+The model assertion (original JSON) may look like this:
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+Note that this model assertion does not have a `system-user-authority` field. When a model assertion lacks this key, a system-user assertion can only be signed by a key registered to the SSO account specified by the `brand-id` field. But, in this case, you do have the key needed to sign the system-user assertion.
+
+### Specifying other SSO accounts that can sign the system-user assertion
+
+You can use the optional `system-user-authority` field to list a set of SSO account IDs that are authorized to sign system-user assertions for any image built with this model assertion.
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "system-user-authority": [
+    "ANOTHER-SSO-ACCOUNT-ID",
+    "YET-ANOTHER-SSO-ACCOUNT_ID"
+  ],
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+In this case, any one who can log in to either ANOTHER-SSO-ACCOUNT-ID or YET-ANOTHER-SSO-ACCOUNT_ID accounts can sign system-user assertions with keys registered to those accounts. And such system-user assertions are valid for systems built with this model.
+
+### Self-signed system-user assertions
+
+A model can also specify that anyone with a registered key can create a valid system-user assertion. This is done with an asterisk ("*") in the `system-user-authority` field:
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "system-user-authority": "*",
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+## Creating a system-user assertion
+
+We now know that creating a valid system-user assertion is limited by the model assertion. And, there is information in the model assertion that is needed to create the system-user assertion, specifically the `brand-id` and the `model`.
+
+
+There's a snap that makes this part easy: make-system-user
+
+1. Install it 
```

Comment:
it can be installed on a core system that has the classic snap installed too. I am  not sure this topic is the place to explain classic though. I've add brief text saying its for classic envs only.